torch_glow/src/PyTorchCommon.cpp

```diff
@@ -227,7 +216,26 @@ at::Tensor glowTypeToEmptyPTTensor(const glow::Type &glowType) {
 }
 
 glow::Tensor ptTensorToGlowTensor(const at::Tensor &ptTensor) {
-  auto glowType = ptTypeToGlowType(*c10::TensorType::create(ptTensor));
+  auto ptType = *c10::TensorType::create(ptTensor);
+
+  DCHECK(ptType.scalarType().has_value())
+      << "TensorType has no associated scalar type.";
+  const auto concreteSizes = ptType.sizes().concrete_sizes().value();
+  std::vector<size_t> dims;
+  for (const auto &size : concreteSizes) {
+    dims.push_back(static_cast<size_t>(size));
+  }
+
+  auto scalarType = ptType.scalarType().value();
+  auto glowElemKind = scalarTypeToElemKind(scalarType);
+
+  glow::Type glowType;
+  if (ptTensor.is_quantized()) {
```

Comment:
We should probably check that it's not channel wise quantized right?

Reply:
agree, let me take a look.